ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -321,6 +321,12 @@ <h3> Monitoring Dashboards:</h3>
             </li>
       </span>
       <span *ngIf="isSharded">
+        <span *ngIf="resourcesGenerated.AggMonitoringDashboardName !== '' && isSharded">
+          <li>
+            Aggregated Monitoring Dashboard:
+            <a [href]="resourcesGenerated.AggMonitoringDashboardUrl" target="_blank">{{resourcesGenerated.AggMonitoringDashboardName}}</a>
+          </li>
+        </span>
```

Comment:
Nit: Since you have now added a new section for generated resources, I think we should display them in a tabular format 🙂